setup.py

```diff
@@ -43,22 +44,12 @@ def has_environment_marker_support():
     try:
         return pkg_resources.parse_version(setuptools.__version__) >= pkg_resources.parse_version('0.7.2')
```

Comment:
This should check if the comparison is False and raise the same `SystemExit` below; returning `False` to the caller has no effect after all.